src/sql/workbench/parts/notebook/cellViews/output.component.ts

```diff
@@ -70,12 +70,14 @@ export class OutputComponent extends AngularDisposable implements OnInit {
 		}
 	}
 
-	private renderOutput() {
+	private renderOutput(focusAndScroll?: boolean) {
```

Comment:
Not a huge deal, but I find it better (clearer) to just use a default instead

```
renderOutput(focusAndScroll: boolean = false) {
```

Reply:
fixed.